app/views/brexit_checker/_detail_text.html.erb

```diff
@@ -0,0 +1,7 @@
+<% unless detail_text.nil? %>
+  <%= render "govuk_publishing_components/components/details", {
+    title: detail_title  || "Why am I being asked this question?"
```

Comment:
@vanitabarrett thanks for this idea, i think it makes a lot of sense.
Does this look good to you right now?